robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -22,6 +22,10 @@ const ROSWARN = 4  // warning level
 const ROSERROR = 8 // error level
 const ROSFATAL = 16 // fatal/critical level
 
+
+// logs below this level will not be published or printed. Issue #202 will allow the user to set this value
+const MINIMUM_LOG_LEVEL = 2 
```

Comment:
Actually, I should use the constant ROSINFO here but yeah

Reply:
```suggestion
const MINIMUM_LOG_LEVEL = ROSINFO 
```